src/pages/lessons/[slug].tsx

```diff
@@ -342,18 +339,6 @@ const Lesson: FunctionComponent<LessonProps> = ({initialLesson}) => {
     }
   }, [currentPlayerState])
 
-  React.useEffect(() => {
-    const handleRouteChange = () => {
-      if (!autoplay) {
```

Comment:
this `!autoplay` was the culprit here and we were never entering the load state. Moved this below into the effect that monitors for a slug change (same idea) and send the load and initial lesson then